recipe/meta.yaml

```diff
@@ -1,34 +1,42 @@
-{% set version = "6.0.1" %}
+{% set name = "envisage" %}
+{% set version = "7.0.3" %}
+{% set python_min = "3.9" %}
```

Comment:
This should not be needed.
```suggestion
```

Reply:
In https://github.com/enthought/envisage/blob/main/pyproject.toml `importlib-resources` is required for `python<3.9`. To avoid this I added `python_min`. Why is this not needed?

Reply:
Because this is the default/global pin. You should only set the variable if you need a Python higher than 3.9.

Reply:
Thanks, done!